### PR TITLE
The astroid now starts in the middle.

### PR DESCRIPTION
**Pull request by Ashthecool**
The asteroids now start in the middle.

### CODE REVIEW

File: my_game.py [hunk 1]
```diff
@@ -387,7 +390,10 @@ def on_update(self, delta_time):
         self.asteroid_list.update()
 
         # Temporary Asteroid test
-        new_asteroid = Asteroid()
+        new_asteroid = Asteroid(
+            center_x=SCREEN_WIDTH//2,
+            center_y=SCREEN_HEIGHT//2
+        )
         self.asteroid_list.append(new_asteroid)
```
Comment (tobiasbp) [inline my_game.py:397]: This is a bug from a previous commit. Please fix it. We don't want to add an Asteroid on every frame :scream:

**Review by tobiasbp — CHANGES_REQUESTED** [review]

File: my_game.py (hunk 1 above)
Comment (Ashthecool) [reply my_game.py:397]: Alright!

Comment (Ashthecool) [reply my_game.py]: I fixed it. The problem was that it was always updating to be the same as change_x and change_y which both is 0.

File: my_game.py [hunk 2]
```diff
@@ -69,7 +69,7 @@ def update(self):
 
 class Asteroid(arcade.Sprite):
     
-    def __init__(self, center_x=0, center_y=0):
+    def __init__(self, center_x=50, center_y=50):
```
Comment (tobiasbp) [inline my_game.py:72]: Let's remove the default values here. So: `def __init__(self, center_x, center_y):`

**Review by tobiasbp — CHANGES_REQUESTED** [review]

File: my_game.py (hunk 1 above)
Comment (tobiasbp) [reply my_game.py]: Good catch!

File: my_game.py (hunk 2 above)
Comment (Ashthecool) [reply my_game.py:72]: alright

**Review by tobiasbp — APPROVED** [review]

**Pull request merged**